### ISSUE TITLE
Documents List | Add strong tag for read/unread

### ISSUE THREAD

**Issue by gnakm** [department-of-veterans-affairs/caseflow#2233]
**Description**
Currently unread document types are tagged as `<b>` when it should be `<strong>`. This will be helpful for users who use screen readers. 

## AC
- Change `<b>` tag to `<strong>` for unread messages (bold).

**Comment by mogrenAtWork** [comment]
@gnakm did you want the unread items in the list to stay bold? `<em>` will italicize them, `<strong>` will keep the item bold but will be useful for screen readers.

**Comment by gnakm** [comment]
@mogrenAtWork unread should be bold 😄

**Comment by astewarttistatech** [comment]
**PASSED**

Validated in: UAT
Browsers: FF and Chrome

Steps to Validate: 
1. Access Reader
2. In the Document List view, "inspect" a document that hasn't been viewed yet (should appear bold in type)
3. Verify that it's tag is <strong> instead of <b> for bold. 

Screenshots: 
<img width="1295" alt="screen shot 2017-06-20 at 11 33 05 am" src="https://user-images.githubusercontent.com/23038505/27341840-71359466-55ac-11e7-8c53-cc95add75921.png">
